lite/examples/image_classification/ios/ImageClassification/ViewControllers/InferenceViewController.swift

```diff
@@ -65,7 +65,7 @@ class InferenceViewController: UIViewController {
   private let bottomSpacing: CGFloat = 21.0
   private let minThreadCount = 1
   private let bottomSheetButtonDisplayHeight: CGFloat = 44.0
-  private let infoTextColor = UIColor.black
+  private let infoTextColor = UIColor(named: "darkOrLight")
```

Comment:
Just realized that this doesn't compile on iOS 10 and lower, because the `UIColor(named:)` init method is only available from iOS 11. Could you make a few changes to make it compatible with lower iOS versions? I think this could be done in a few steps.

1) Change `infoTextColor` to a private var (and move this line to the private vars block below) and keep the original default value of `UIColor.black`.
```swift
private var infoTextColor = UIColor.black
```

2) In the `viewDidLoad()` override method:

```swift
// Set the info text color on iOS 11 and higher.
if #available(iOS 11, *) {
  infoTextColor = UIColor(named: "darkOrLight")!
}
```

Then I think this should still build and run fine for older iOS versions.